### PR TITLE
Improve summaries

### PR DESCRIPTION
**Pull request by Pfeil**
In TF D&D, we discussed what makes summaries great, and we looked at the services "teaser texts". I included some discussed proposals, and more are likely going to come from the group.

**Feel free to discuss them here and improve the texts, even though I marked it as a draft so it does not get merged yet**. We would like to derive general recommendations for summary-writing from this. Thanks. :)

### CODE REVIEW

File: index.markdown [hunk 1]
```diff
@@ -15,7 +15,7 @@ target="base-repo/index.html"
 background="assets/images/disks.jpg"
 title="base-repo"
 subtitle="A place to be for your data"
-content="The base-repo is a generic, general purpose research data repository service offering clear, machine-actionable RESTful interfaces for storing, retrieving and managing research data."
+content="The base-repo is a generic, general purpose research data repository service. It is offering clear, machine-actionable RESTful interfaces for storing, retrieving and managing research data."
```
Comment (VolkerHartmann) [inline index.markdown:18]: It offers a clear...?

**Review by VolkerHartmann — APPROVED** [review]
Looks fine to me.

File: index.markdown [hunk 2]
```diff
@@ -15,15 +15,15 @@ target="base-repo/index.html"
...
 %}
 
 {% include card.html tags="metadata repository restful"
 target="metastore/index.html"
 background="assets/images/metadata.jpg"
 title="MetaStore"
-subtitle="Metadata - for sure"
-content="MetaStore is a research data repository service for managing formally quality-controlled metadata documents based on JSON or XML."
+subtitle="Storing consistent metadata"
+content="MetaStore is a research data repository service storing metadata documents and schemas. By associating each document with a schema and validating it, quality and consistency are maintained. Supports JSON and XML."
```
Comment (ghost) [inline index.markdown]: Suggestions:
1. "... repository service for storing ..."
2. "... validating it, quality and consistency are ensured." Oder ganz umdrehen: "Quality and consistency are ensured by associating and validating each document against a schema." 
3. "It supports JSON and XML."

File: index.markdown [hunk 3]
```diff
@@ -100,8 +100,7 @@ target="fairdoscope/index.html"
 background="assets/images/fairdoscope/logo.png"
 title="FAIR-DOscope"
 subtitle="Explore the facets of FAIR Digital Objects"
-content="FAIR-DOscope is an easy-to-use, generic FAIR Digital Object viewer and browser accepting PIDs of FAIR DOs and presenting
-the associated PID record in a graphical and user-friendly way."
+content="FAIR-DOscope is an easy-to-use, generic FAIR Digital Object viewer and browser. It accepts PIDs of FAIR DOs and presents the associated PID record in a graphical and user-friendly way."
```
Comment (ghost) [inline index.markdown]: Suggestion: "It accepts PIDs of FAIR DOs and visualizes the associated PID record in a user-friendly way."

File: index.markdown [hunk 4]
```diff
@@ -15,15 +15,15 @@ target="base-repo/index.html"
...
+content="The base-repo is a generic, general purpose research data repository service. It is offering clear, machine-actionable RESTful interfaces for storing, retrieving and managing research data."
 %}
 
 {% include card.html tags="metadata repository restful"
 target="metastore/index.html"
 background="assets/images/metadata.jpg"
 title="MetaStore"
-subtitle="Metadata - for sure"
-content="MetaStore is a research data repository service for managing formally quality-controlled metadata documents based on JSON or XML."
+subtitle="Storing consistent metadata"
```
Comment (ghost) [inline index.markdown]: Possible alternatives:
1. Store consistent metadata
2. Store metadata consistently

**Review by ghost — COMMENTED** [review]
I have added some suggestions for rephrasing.

File: index.markdown [hunk 5]
```diff
@@ -75,7 +75,7 @@ on how to appoach different concepts related to FAIR Digital Objects, i.e., Data
 target="metadatahub/index.html"
 background="assets/images/turntable.png"
 title="MetadataHub"
-subtitle="One to serve all"
+subtitle="A hub for repositories"
```
Comment (ghost) [inline index.markdown]: A possible alternative might be "An access hub for repositories"

File: index.markdown (hunk 1 above)
Comment (Pfeil) [reply index.markdown:18]: I tried to keep the plural as I tried not to change the meaning. Should I change it?
I could make "It offers clear, machine, actionable RESTful interfaces for ...", keeping the plural and remove the -ing-form. I like the simplification. But before that, should it be described as a single interface or multiple interfaces?

Comment (ghost) [reply index.markdown:18]: If multiple interfaces are offered, I would go with your revised phrasing.

File: index.markdown (hunk 5 above)
Comment (Pfeil) [reply index.markdown]: +1 Access hub describes it even better, I think.

File: index.markdown (hunk 2 above)
Comment (Pfeil) [reply index.markdown]: I'd vote for 2+3. Thoughts regarding 1:

- It's ok, but I think a comma between service and storing would be enough.
- Service is such a general term and still sounds like a hosted/cloud solution. Maybe replace with software?

File: index.markdown (hunk 3 above)
Comment (Pfeil) [reply index.markdown]: I applied it.

File: index.markdown (hunk 4 above)
Comment (Pfeil) [reply index.markdown]: I applied 2 :)

File: index.markdown (hunk 2 above)
Comment (ghost) [reply index.markdown]: I rephrased it and replaced service by software. See commit [d30e48b](https://github.com/kit-data-manager/webpage/pull/25/commits/d30e48bd68f611d8c2aa1ad061fdb863f0480212).

**Review by ThomasJejkal — APPROVED** [review]

**Review by ThomasJejkal — APPROVED** [review]

**Review by ThomasJejkal — APPROVED** [review]

**Review by ThomasJejkal — APPROVED** [review]

File: index.markdown (hunk 1 above)
Comment (ThomasJejkal) [reply index.markdown:18]: Stay with the plural, that's fine. Alternatively, one could remove RESTful as there is also the possibility to enable an OAI-PMH interface, which is not RESTful, but which justifies the plural even if all endpoints belonging to base-repo are interpreted as one interface. ;-)

**Review by ghost — APPROVED** [review]

**Pull request merged**